index.html

```diff
@@ -32,16 +32,30 @@
         <nav class="toc">
           <h1>Contents</h1>
           <ul>
-            <li><a href="#what-are-material-icons">What are material icons</a></li>
+            <li><a href="#what-are-material-icons-">What are material icons</a></li>
             <li><a href="#getting-icons">Getting the icons</a></li>
             <li><a href="#icon-font-for-the-web">Icon fonts for the web</a></li>
             <li><a href="#icon-images-for-the-web">Icon images for the web</a></li>
             <li><a href="#icons-for-android">Icons for Android</a></li>
             <li><a href="#icons-for-ios">Icons for iOS</a></li>
+            <li><a href="#icons-in-rtl">Icons in RTL</a></li>
           </ul>
         </nav>
 
-        <h1 id="what-are-material-icons-">What are material icons?</h1>
+        <style type="text/css">
```

Comment:
Moved it into the header section. I do not want to move it into the CSS file as it is being generated.